src/library_nodefs.js

```diff
@@ -123,15 +119,6 @@ addToLibrary({
         var stat;
         NODEFS.tryFSOperation(() => stat = fs.lstatSync(path));
         if (NODEFS.isWindows) {
-          // node.js v0.10.20 doesn't report blksize and blocks on Windows. Fake
```

Comment:
We do still support MIN_NODE_VERSION=101900 (10.19.0) so if we want to drop this I guess we would need to bump that minimum too?

Reply:
Oh wait, thats is 0.10.20!

Reply:
Can you add some stat tests to `test-node-compat` in `.circleci/config.yml`?